src/btree/bt_cursor.c

```diff
@@ -2232,6 +2361,7 @@ __cursor_range_selectivity_do_traversal_step(WT_ITEM *srch_key, WT_SESSION_IMPL
   WT_COLLATOR *collator, WT_CURSOR_BTREE *cbt, WT_PAGE *page, WT_PAGE_INDEX *parent_pindex,
   WT_PAGE_INDEX *pindex, WT_REF *current, WT_REF **descent_ptr, uint32_t *indx_ptr)
 {
+    WT_DECL_RET;
```

Comment:
I see this above too, what does it do?

Reply:
It declares the "ret" variable, which we need for `WT_TRET`  (Set "ret" if not already set) and `return (ret)`. These are used all over the code, so that's why I'm using them here